SwiftChat/ModelLoader.swift

```diff
@@ -14,7 +14,11 @@ class ModelLoader {
     static let lastCompiledModel = models / "last-model.mlmodelc"
         
     static func load(url: URL?) async throws -> LanguageModel {
-        if let url = url {
+        defer {
+            url?.stopAccessingSecurityScopedResource()
+        }
+        
+        if let url = url, url.startAccessingSecurityScopedResource() {
```

Comment:
```suggestion
        if let url = url, url.startAccessingSecurityScopedResource() {
            defer {
                url?.stopAccessingSecurityScopedResource()
            }
```

Just a minor thing, but I'm not sure it's ok to call `stopAccessingSecurityScopedResource` if `start...` failed.